test/tests/nek_errors/invalid_field/nek_uo.i

```diff
@@ -19,10 +19,10 @@
 [UserObjects]
   [subchannel_binning]
     type = HexagonalSubchannelBin
-    bundle_pitch = 0.02583914354890463
```

Comment:
Why were these changed? We should avoid changing any test specifications unless we absolutely need to.

Reply:
Now that the temperature check was moved to within `solutionPointer`, running that test with the original `HexagonalSubchannelBins` inputs throws an error saying the bins set are not valid, which is an error reached before getting to `solutionPointer` to throw the temperature error. 

Since this test is for checking an invalid temperature field, I changed the `HexagonalSubchannelBins` inputs to these new values so that the code can run enough to reach the `solutionPointer` and throw the temperature error we want to see. Otherwise, this test would throw an error related to `HexagonalSubchannelBins`

Reply:
Oh, gotcha - I didn't realize that invalid values were here in the first place. Thanks for the fix!